index.html

```diff
@@ -1,30 +1,100 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css" />
   </head>
-
```

Comment:
remove spaces

Reply:
- [ ] Kindly make sure to deliver politely, and well-explained review comments so that students understand the point 🙏 😄 

_Kindly see the first comment for more clarification._

Reply:
- [ ] you should avoid spaces in your code to make code look professional

Reply:
- [ ] Dear @Mucyosoda, I think you should make the changes requested by the reviewer pointed out in the above comment. Maybe, thanks, is not the right place here. After this review, you should put the changes you make right below this comment to address this review comment I give you.